### PR TITLE
Add nvmrc and update travis' chrome reference

### PR DESCRIPTION
**Pull request by gesinger**
## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/liwecukasi/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Comment by forbesjo** [comment]
You may need `sudo: required`

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: false
 dist: trusty
 language: node_js
 node_js:
-  - 'node'
+  - lts/*
```
Comment (forbesjo) [inline .travis.yml]: You can remove `node_js:` since this is already defined in the .nvmrc

**Comment by squarebracket** [comment]
Not sure if these fixes will stop errors like [this](https://travis-ci.org/videojs/videojs-contrib-hls/jobs/330985514), but I've had sporadic disconnect errors in the past when testing. Maybe we should add `browserDisconnectTolerance: 3` to the karma config?

**Comment by squarebracket** [comment]
Wow, what a roller coaster. Hopefully you still have some hair left cuz I’d have pulled all of mine out.

**Comment by gesinger** [comment]
Yeah @squarebracket , it's been a fun one. Still have one test to fix, but 12/13 have been resolved. Hopefully have enough hair left for that one, but it may be close.

**Pull request merged**